signature-pad/src/main/java/com/github/gcacace/signaturepad/views/SignaturePad.java

```diff
@@ -208,7 +208,10 @@ public boolean onTouchEvent(MotionEvent event) {
             case MotionEvent.ACTION_DOWN:
                 getParent().requestDisallowInterceptTouchEvent(true);
                 mPoints.clear();
-                if (isDoubleClick()) break;
+                if (isDoubleClick()) {
```

Comment:
Can you extract from the function `isDoubleClick()` the part where the signature is cleared, so we can have the clearing part and the `setIsEmpty(true)` in the same place? I'd like to have `isDoubleClick()` as immutable method.

Reply:
Sure. Can you look at the new commit to check if that's what you wanted?